src/scheduler.js

```diff
@@ -162,8 +162,14 @@ class Scheduler extends events.EventEmitter {
       errJson = `(error formatting JSON: ${e})`;
     }
 
-    this.notify.email({
-      address: email,
+    let email = this.createEmail(hook, err, errJson);
+    this.notify.email(email);
+    this.notify.lastEmail = _.merge({}, email, {err, errJson});
```

Comment:
For the error and JSON, I think we can assert that some part of those appear in the content using a regular expression.

I don't think `this` would have worked, actually, since this isn't a class.  But my knowledge of how JS classes work is a little spotty.